docs/external-node.md

```diff
@@ -334,6 +338,79 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
       sudo systemctl start antrea-agent
       ```
 
+##### Container Installation
+
+1. The Docker image can be built from source code or can be downloaded from the antrea repository.
+
+   1. Source Code
+   
+      Build `antrea-ubuntu` docker image in the root of the antrea code tree.
+
+      ```bash
+      make
+      ```
+
+      Rename that image to `projects.registry.vmware.com/antrea/antrea-ubuntu:latest`
+
+      ```bash
+      docker image tag <image-id> projects.registry.vmware.com/antrea/antrea-ubuntu:latest
+      ```
+
+      Copy the `projects.registry.vmware.com/antrea/antrea-ubuntu:latest` image to install VM. Please follow the below steps.
+
+      ```bash
+      # Save it in a tar file
+      docker save -o <tar file path in source host machine> projects.registry.vmware.com/antrea/antrea-ubuntu
+ 
+      # Copy this tar file using simple tools such as cp, rsync, scp, etc, or any other method that you prefer.
+      # Then load that image in the install VM.
+      docker load -i <path to image tar file>
+      ```
+
+   2. Repository
+   
+      The released version of `antrea-ubuntu` Docker image can be downloaded from this `projects.registry.vmware.com/antrea/antrea-ubuntu` 
+      repository. Pick a version from the [list of release](https://github.com/antrea-io/antrea/releases). For any given
+      release `<TAG>` (e.g. `v0.1.0`), download `antrea-ubuntu` Docker image as follows:
+
+      ```bash
+      docker pull "projects.registry.vmware.com/antrea/antrea-ubuntu:<TAG>"
+      ```
+
+      The [installation script](../hack/externalnode/install-vm.sh) automatically downloads the specific released 
+      version of `antrea-ubuntu` docker image on installation VM by mentioning the argument `--antrea-version` in a command. 
+      Also, the script automatically loads that image into the Docker.For any given release `<TAG>` (e.g. `v0.1.0`).
+      
+      ```bash
+         --antrea-version <TAG>
+      ```
+      
+      Note: If the `--antrea-version` argument is not mentioned, The latest version of `antrea-ubuntu` Docker image can be 
+      downloaded from this `projects.registry.vmware.com/antrea/antrea-ubuntu` repository automatically.
+
+2. Copy configuration files to the VM, including [antrea-agent.conf](../build/yamls/externalnode/conf/antrea-agent.conf),
+   which specifies agent configuration parameters;
+   `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig`, which were
+   generated in steps 4 and 5 of [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster).
+
+3. Bootstrap `antrea-agent` using the [installation script](../hack/externalnode/install-vm.sh) 
+   as shown below (Ubuntu 18.04 and 20.04, Rhel 8.4).
+
+   ```bash
```

Comment:
You can keep only one of the usage. No need to highlight all the options.
Suggest to keep L408-L412, and remove L399 to L405.